presto-product-tests/bin/run_on_docker.sh

```diff
@@ -70,25 +70,46 @@ function stop_application_runner_containers() {
   done
 }
 
+function stop_all_containers() {
+  local ENVIRONMENT
+  for ENVIRONMENT in $(getAvailableEnvironments)
+  do
+     stop_docker_compose_containers
+  done
+}
+
 function stop_docker_compose_containers() {
```

Comment:
I think it is better to pass explicitly the environment you want to stop the containers for.